packages/common/src/lib/common/tools.ts

```diff
@@ -2609,6 +2613,12 @@ export function formatAliasValue(
     }
 
     if (targetObj?.alias?.read) {
+        if (!sourceObj) {
+            logger.error(
+                `${logNamespace} source in "${targetId}" not exist for "read" function: "${targetObj.alias.read}"`
```

Comment:
```suggestion
                `${logNamespace}source in "${targetId}" does not exist for "read" function: "${targetObj.alias.read}"`
```

Reply:
And formally the "space too much after ${logNamespace} needs to be changed in whole function :-)